backends-common/postgres/src/main/java/org/apache/james/backends/postgres/PostgresTableManager.java

```diff
@@ -105,10 +105,14 @@ public Mono<Void> alterTableEnableRLS(PostgresTable table) {
     }
 
     private String rowLevelSecurityAlterStatement(String tableName) {
-        return "SET app.current_domain = ''; ALTER TABLE " + tableName + " ADD DOMAIN varchar(255) not null DEFAULT current_setting('app.current_domain')::text;" +
```

Comment:
Can just `ADD COLUMN IF NOT EXISTS domain varchar(255)...` IMO.
https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-DESC-ADD-COLUMN

no need to use a custom function here IMO.

Reply:
> Can just ADD COLUMN IF NOT EXISTS domain varchar(255)... IMO.

I did it in this commit.

Reply:
Ok. Looks good to me.